plugins/invisible-typing/src/Settings.tsx

```diff
@@ -16,7 +16,7 @@ export function Settings(): React.ReactElement {
       </SwitchItem>
       <SwitchItem
         {...util.useSetting(cfg, "invisible", true)}
-        note="When enabled, invisibility applies to all channels. When disabled, invisibility is channel based.">
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_clarification):** Clarification of setting description enhances user understanding.

```suggestion
note="Invisibility can be applied globally to all channels or individually per channel. When enabled globally, this setting acts as the default behavior for each channel's invisibility state.">
```